index.html

```diff
@@ -1355,12 +1359,18 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
     group discussion.
   </p>
   <section>
-    <h2>Exposing user information</h2>
+    <h3>Exposing user information</h3>
     <p>
       The <a>user agent</a> should never share information about the user to the web page
       (such as the shipping address) without user consent.
     </p>
   </section>
+  <section>
+    <h3>Incognito Mode</h3>
+    <ul>
+      <li>This API should remain available in a browser's "incognito mode".</li>
+      <li>Browsers that offer "persistent permission" features for sharing user payment information automatically with merchants should ignore those settings in icognito mode and require explicit user consent.</li>
```

Comment:
Persistent permissions should link to the Permission spec. I think it defines what that is.